zebrad/src/commands.rs

```diff
@@ -38,6 +38,8 @@ pub enum ZebradCmd {
 }
 
 impl ZebradCmd {
+    pub const SHA: &'static str = env!("VERGEN_SHA_SHORT");
```

Comment:
Git uses `SHA1`s for a bunch of different objects. This is the `SHA1` of the git `HEAD` commit:
```suggestion
    pub const GIT_COMMIT: &'static str = env!("VERGEN_SHA_SHORT");
```